GameServer/Logic/Config.cs

```diff
@@ -33,7 +33,7 @@ private void LoadConfig(string path)
             foreach (var player in playerConfigurations)
             {
                 var playerConfig = new PlayerConfig(player);
-                Players.Add(string.Format("player{0}", Players.Count + 1), playerConfig);
+                Players.Add($"player{Players.Count + 1}", playerConfig);
```

Comment:
But if you're going to do it that way, you could as well do
```CS
var playerString = string.Format("player{0}", Players.Count + 1);
Players.Add(playerString, playerConfig);
```
Choose which you prefer more